hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/SchemaVersionTableDefinition.java

```diff
@@ -37,9 +39,12 @@
 @Singleton
 public class SchemaVersionTableDefinition implements ReconSchemaDefinition {
 
+  private static final Logger LOG = LoggerFactory.getLogger(SchemaVersionTableDefinition.class);
+
   public static final String SCHEMA_VERSION_TABLE_NAME = "RECON_SCHEMA_VERSION";
   private final DataSource dataSource;
   private DSLContext dslContext;
+  private int latestSLV;
```

Comment:
This field not requried, can be obtained by calling ReconLayoutFeature method.

Reply:
The latestSLV field is needed to keep the recon and recon-codegen packages independent. Directly accessing methods from the recon package in recon-codegen would create unnecessary dependency. Attila has advised not to do this